layers/+lang/ipython-notebook/packages.el

```diff
@@ -157,4 +158,11 @@
         ("9" ein:notebook-worksheet-open-last)
         ("+" ein:notebook-worksheet-insert-next)
         ("-" ein:notebook-worksheet-delete)
-        ("x" ein:notebook-close)))))
+        ("x" ein:notebook-close))
+      (spacemacs/set-leader-keys "ein" 'spacemacs/ipython-notebook-transient-state/body))))
+
+(defun ipython-notebook/init-ob-ipython ()
+  (use-package ob-ipython
+    :defer t
```

Comment:
Oops, my mistake. I had some other settings that were causing problems. Sorry for the noise and thanks again for pushing this forward. I hope we get it into spacemacs soon.